pkgs/top-level/all-packages.nix

```diff
@@ -26669,6 +26669,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Cocoa;
   };
 
+  furtherance = callPackage ../applications/misc/furtherance {  };
```

Comment:
Use one space between attrset braces.

```suggestion
  furtherance = callPackage ../applications/misc/furtherance { };
```

Reply:
Donion rings! 
I'm sorry for that typo, didn't even noticed when I typed it.